packages/react-charting/src/utilities/colors.ts

```diff
@@ -0,0 +1,242 @@
+type Palette = { [key: string]: string[] };
+
```

Comment:
Add a comment that 0 represents light mode and 1 represents dark mode

Reply:
done